src/name_bazaar/ui/components/share_buttons.cljs

```diff
@@ -1,8 +1,30 @@
-(ns name-bazaar.ui.components.share-buttons)
+(ns name-bazaar.ui.components.share-buttons
+  (:require [re-frame.core :refer [reg-sub subscribe]]
+            [reagent.ratom :refer-macros [reaction]]
+            [clojure.string :as cstr]
+            [name-bazaar.ui.utils :refer [path-for]]))
 
 (defn share-buttons []
-  [:div.share-buttons
-   [:div.title "Share On:"]
-   [:i.icon.fb]
-   [:i.icon.twitter]])
+  (let [root-url (subscribe [:root-url])
+        my-address (subscribe [:district0x/active-address])
+        title "My ENS Offerings"
+        page-url (reaction (cstr/replace
```

Comment:
Please use normal `subscribe` for consistency. Also you must check now if we're using HTML5 push state urls or with #, to share url correctly. See `district0x.ui.utils/hashroutes?`

Reply:
Actually no need to detect this.`https://namebazaar.io` will always have urls without hashtag